apps/prairielearn/elements/pl-drawing/pl-drawing.mustache

```diff
@@ -25,7 +25,14 @@
 {{^parse_error}}
 <div id="pl-drawing-{{uuid}}" class="pl-drawing-container pl-requires-svg-mathjax">
     <div>
-        <canvas width="{{width}}" height="{{height}}"> </canvas>
+        <canvas
+            width="{{width}}"
+            height="{{height}}"
+            {{#preview_mode}}role="img"{{/preview_mode}}
+            {{#aria_label}}aria-label="{{aria_label}}"{{/aria_label}}
+        >
+            {{^preview_mode}}<p>Your browser does not support the canvas element, and this drawing needs canvas interaction.</p>{{/preview_mode}}
```

Comment:
Canvas is supported by almost 98% of browsers: https://caniuse.com/canvas

That's actually higher than the percentage of browsers that support `const` in JS, without with most of our scripts completely fail in the browser: https://caniuse.com/canvas

I don't think this change does anything useful for us. People using IE 8 should just not.

Reply:
Changed in https://github.com/PrairieLearn/PrairieLearn/pull/11867/commits/2aaedffecb7da9723b5ee7184ca98bc2c66e942d

Reply:
Presumably you meant f1039d4? 🙏